include/wx/gtk/textentry.h

```diff
@@ -99,7 +99,7 @@ class WXDLLIMPEXP_CORE wxTextEntry : public wxTextEntryBase
     // Call this from the overridden wxWindow::GTKIMFilterKeypress() to use
     // GtkEntry IM context.
     int GTKEntryIMFilterKeypress(GdkEventKey* event) const;
-
+    int m_maxlen;
```

Comment:
Should have a comment, should be initialized, almost certainly should not be defined in between 2 functions without any blank lines around it. Please try to keep things logically organized and more or less readable.

Oh, and it should also be initialized in the declaration.